### PR TITLE
AF-799

### PR DESCRIPTION
**Pull request by desmax74**
Changes for the new maven compiler, it includes Takari with our changes (1.13.5) ,  the changes are only on the maven compiler and not the integration of the workbench

### CODE REVIEW

**Comment by desmax74** [comment]
@jhrcek @porcelli

File: guvnor-bom/guvnor-bom.iml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jhrcek) [inline guvnor-bom/guvnor-bom.iml]: @desmax74 there's a [general recommendation](https://intellij-support.jetbrains.com/hc/en-us/articles/206544839-How-to-manage-projects-under-Version-Control-Systems) to add `*.iml` files to .gitignore. 

Is there any reason to include this particular iml file into git?

Comment (desmax74) [reply guvnor-bom/guvnor-bom.iml]: Isn't a manual addition, let me check why this is here

Comment (desmax74) [reply guvnor-bom/guvnor-bom.iml]: Found the reason, going to delete

**Comment by desmax74** [comment]
Jenkins retest this

**Pull request merged**